src/explore-education-statistics-common/src/modules/charts/components/ChartRenderer.tsx

```diff
@@ -18,7 +18,12 @@ import React, { memo, useMemo } from 'react';
 
 export type ChartRendererProps = {
   source?: string;
+  releaseId?: string;
+  dataBlockParentId?: string;
```

Comment:
As discussed - if the callback approach works, we shouldn't need these parameters